OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -295,12 +300,19 @@ protected override IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr,
 
 			protected override string GetCursor(World world, CPos cell, int2 worldPixel, MouseInput mi)
 			{
-				return "ability";
+				return cursor;
 			}
 		}
 
 		class BeginMinefieldOrderTargeter : IOrderTargeter
 		{
+			readonly string startMinefieldCursor;
```

Comment:
nit: start vs begin, do we need to use 2 separate terms?

Reply:
Shortened it.